HLTrigger/Configuration/python/customizer_AlpakaPixelTracking_Phase2_141X.py

```diff
@@ -0,0 +1,247 @@
+            maxChi2 = cms.double(5.0),
+            minPt   = cms.double(0.9),
+            maxTip  = cms.double(0.3),
+            maxZip  = cms.double(12.),
+        ),
+        # autoselect the alpaka backend
+        alpaka = cms.untracked.PSet(
+            backend = cms.untracked.string('')
+        )
+    )
+    process.hltPhase2PixelTracks = cms.EDProducer("PixelTrackProducerFromSoAAlpakaPhase2",
+        beamSpot = cms.InputTag("hltOnlineBeamSpot"),
+        minNumberOfHits = cms.int32(0),
+        minQuality = cms.string('tight'),
+        pixelRecHitLegacySrc = cms.InputTag("siPixelRecHits"),
+        trackSrc = cms.InputTag("hltPhase2PixelTracksSoA")
+    )
+    process.hltPhase2PixelTracksSequence = cms.Sequence(
+         process.HLTBeamSpotSequence
+        +process.hltPhase2PixelTracksAndHighPtStepTrackingRegions # needed by highPtTripletStep iteration
+        +process.hltPhase2PixelFitterByHelixProjections # needed by tracker muons
+        +process.hltPhase2PixelTrackFilterByKinematics  # needed by tracker muons
+        +process.hltPhase2PixelTracksSoA
+        +process.hltPhase2PixelTracks
+    )
+    return process
+
+def AlpakaHLTFullTracking(process):
+    # original cfg at hand as in 14_1_0_pre4 simplified menu 
+    # https://lguzzi.web.cern.ch/lguzzi/Tracking/Phase2/hlt-dump_simplifiedMenu_14_1_0_pre4.txt
+    ''' track building seeded by hltPhase2PixelTracks. This configuration runs the initialStep
+    and the highPtTripletStep
+    '''
+    process.HLTTrackingV61Sequence = cms.Sequence(
+         process.itLocalRecoSequence
+        +process.otLocalRecoSequence
+        +process.trackerClusterCheck
+        +process.hltPhase2PixelTracksSequence
+        +process.hltPhase2PixelVertices
+        +process.initialStepSequence
+        +process.highPtTripletStepSequence
+        +process.generalTracks
+    )
+    return process
+
+def AlpakaHLTTrackingPath(process):
+    ''' Create a tracking-only path
+    '''
+    process.HLT_AlpakaTrackingPath = cms.Path()
+    process.HLT_AlpakaTrackingPath.insert(item=process.HLTBeginSequence         , index=len(process.HLT_AlpakaTrackingPath.moduleNames()))
+    process.HLT_AlpakaTrackingPath.insert(item=process.HLTTrackingV61Sequence   , index=len(process.HLT_AlpakaTrackingPath.moduleNames()))
+    process.HLT_AlpakaTrackingPath.insert(item=process.HLTEndSequence           , index=len(process.HLT_AlpakaTrackingPath.moduleNames()))
+
+    process.FEVTDEBUGHLTEventContent.outputCommands.append('keep *_siPixelClusters_*_reHLT')
+    process.FEVTDEBUGHLTEventContent.outputCommands.append('keep *_siPixelRecHits_*_reHLT')
+    process.FEVTDEBUGHLTEventContent.outputCommands.append('keep *_hltPhase2PixelTracks_*_reHLT')
+    process.FEVTDEBUGHLTEventContent.outputCommands.append('keep *_generalTracks_*_reHLT')
+    process.FEVTDEBUGHLTEventContent.outputCommands.append('drop IntermediateHitDoublets_hltElePixelHitDoubletsL1Seeded__reHLT')            # remove warning
```

Comment:
Syntax is incorrect - I suppose ```L1Seeded_*_reHLT``` is required here and below?

Reply:
these were leftovers, removed them in 2147f3d41dc5f1f27929683ee5f16bda0dfde052